microxrcedds_cmake_module/package.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>microxrcedds_cmake_module</name>
+  <version>0.5.1</version>
+  <description>Provide CMake module to find eProsima Micro XRCE-DDS.</description>
+  <maintainer email="borjaouterelo@eprosima.com">Borja Outerelo</maintainer>
+  <license>Apache License 2.0</license>
```

Comment:
```suggestion
  <license>Apache-2.0</license>
```
Propose to use the official SPDX identifier from https://spdx.org/licenses/, which eases automated processing of license information.